testutil/src/test/java/org/spine3/test/VerifyShould.java

```diff
@@ -0,0 +1,1312 @@
+            Verify.assertContains(emptyString, nullString);
+        }
+    }
+
+    @Test
+    public void pass_if_string_contains_char_sequence() {
+        Verify.assertContains(emptyString, notEmptyString);
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_is_string_contains_char_sequence() {
+        Verify.assertNotContains(emptyString, notEmptyString);
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_char_sequence_is_null_in_not_contains() {
+        Verify.assertNotContains(null, emptyString);
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_string_is_null_in_not_contains() {
+        Verify.assertNotContains(emptyString, (String) null);
+    }
+
+    @Test
+    public void pass_if_string_not_contains_char_sequence() {
+        Verify.assertNotContains(notEmptyString, emptyString);
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_collection_not_contains_item() {
+        Verify.assertContains(1, Collections.emptyList());
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_collections_is_null_in_contains() {
+        Verify.assertContains(1, (Collection) null);
+    }
+
+    @Test
+    public void pass_if_collection_contains_item() {
+        final Integer item = 1;
+        Verify.assertContains(item, Collections.singletonList(item));
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_immutable_collection_not_contains_item() {
+        Verify.assertContains(1, ImmutableList.of());
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_immutable_collections_is_null_in_contains() {
+        Verify.assertContains(1, null);
+    }
+
+    @Test
+    public void pass_if_immutable_collection_contains_item() {
+        final Integer item = 1;
+        Verify.assertContains(item, ImmutableList.of(item));
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_iterable_not_contains_all() {
+        Verify.assertContainsAll(Collections.emptyList(), 1);
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_iterable_is_null_in_contains_all() {
+        Verify.assertContainsAll(null);
+    }
+
+    @Test
+    public void pass_if_iterable_contains_all() {
+        final Integer item = 1;
+        Verify.assertContainsAll(Collections.singletonList(item), item);
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_map_are_not_equal() {
+        final Map<Integer, Map<Integer, Integer>> firstOne = Collections.singletonMap(1, Collections.singletonMap(1, 1));
+        final Map<Integer, Map<Integer, Integer>> secondOne = Collections.singletonMap(1, Collections.singletonMap(1, 2));
```

Comment:
Check if you fit 120-symbols-per-row restriction. 

One of the smart things you could do was to `static import` the `Collections.singletonMap`. That would make the line shorter. Also the rule is to import the item statically if it is used a lot (and it cannot be confused with something similarly named in the class). 

However, please avoid wildcard imports like `import static foo.bar.Collections.*`.